src/core/qgssqlstatement.h

```diff
@@ -327,23 +327,31 @@ class CORE_EXPORT QgsSQLStatement
       public:
         //! Constructor
         NodeUnaryOperator( QgsSQLStatement::UnaryOperator op, QgsSQLStatement::Node *operand SIP_TRANSFER ) : mOp( op ), mOperand( operand ) {}
-        ~NodeUnaryOperator() override { delete mOperand; }
+        ~NodeUnaryOperator() override {  }
+
+        NodeUnaryOperator( const NodeUnaryOperator &other ) = delete;
+        NodeUnaryOperator &operator=( const NodeUnaryOperator &other ) = delete;
```

Comment:
Personally I tend to place deleted constructors/operators to the private section of the class, for clarity (applies to several similar instances below)

Reply:
Makes sense. Done